src/lib/HST/Options.hs

```diff
@@ -26,6 +36,34 @@ import           HST.Effect.Report              ( Message(Message)
                                                 , reportFatal
                                                 )
 
+-- | A data type for all front ends that can be used for parsing the given input
+--   program in @haskell-src-transformations@.
+data Frontend = HSE | GHClib
+  deriving (Eq, Show)
+
+-- | Name of the @haskell-src-extensions@ front end.
```

Comment:
```suggestion
-- | Name of the @haskell-src-exts@ front end.
```

or maybe even

```suggestion
-- | Name of the 'HSE' front end.
```

If you choose the later, update `ghclib`'s comment accordingly.